src/wanderer.ts

```diff
@@ -46,165 +52,200 @@ export function digitizedMonstersRemaining(): number {
   );
 }
 
-interface ZonePotion {
+interface ZoneUnlocker {
   zone: string;
-  effect: Effect;
-  potion: Item;
+  available: () => boolean;
+  unlocker: Item;
+  noInv?: boolean;
+}
+
+function airportAvailable(element: "spooky" | "stench" | "hot" | "cold" | "sleaze"): boolean {
+  return get(`_${element}AirportToday`) || get(`${element}AirportAlways`);
 }
 
-const zonePotions: ZonePotion[] = [
+const zoneUnlockers: ZoneUnlocker[] = [
   {
     zone: "Spaaace",
-    effect: $effect`Transpondent`,
-    potion: $item`transporter transponder`,
+    available: () => have($effect`Transpondent`),
+    unlocker: $item`transporter transponder`,
   },
   {
     zone: "Wormwood",
-    effect: $effect`Absinthe-Minded`,
-    potion: $item`tiny bottle of absinthe`,
+    available: () => have($effect`Absinthe-Minded`),
+    unlocker: $item`tiny bottle of absinthe`,
   },
   {
     zone: "RabbitHole",
-    effect: $effect`Down the Rabbit Hole`,
-    potion: $item`"DRINK ME" potion`,
+    available: () => have($effect`Down the Rabbit Hole`),
+    unlocker: $item`"DRINK ME" potion`,
+  },
+  {
+    zone: "Conspiracy Island",
+    available: () => airportAvailable("spooky"),
+    unlocker: $item`one-day ticket to Conspiracy Island`,
+    noInv: true,
+  },
+  {
+    zone: "Dinseylandfill",
+    available: () => airportAvailable("stench"),
+    unlocker: $item`one-day ticket to Dinseylandfill`,
+    noInv: true,
+  },
+  {
+    zone: "The Glaciest",
+    available: () => airportAvailable("cold"),
+    unlocker: $item`one-day ticket to The Glaciest`,
+    noInv: true,
+  },
+  {
+    zone: "Spring Break Beach",
+    available: () => airportAvailable("sleaze"),
+    unlocker: $item`one-day ticket to Spring Break Beach`,
+    noInv: true,
   },
 ];
 
-function acceptBestGuzzlrQuest() {
-  if (!Guzzlr.isQuestActive()) {
-    if (
-      Guzzlr.canPlatinum() &&
-      (!Guzzlr.haveFullPlatinumBonus() ||
-        (Guzzlr.haveFullBronzeBonus() && Guzzlr.haveFullGoldBonus()))
-    ) {
-      Guzzlr.acceptPlatinum();
-    } else if (Guzzlr.canGold() && (!Guzzlr.haveFullGoldBonus() || Guzzlr.haveFullBronzeBonus())) {
-      Guzzlr.acceptGold();
-    } else {
-      Guzzlr.acceptBronze();
-    }
-  }
-}
+function testZoneAndUsePotionToAccess(zone: Location | null) {
```

Comment:
I get why you're doing this, but I feel like letting this take in a null input is a bad idea